ragnarok_packets/src/lib.rs

```diff
@@ -1460,6 +1460,13 @@ pub struct MovingEntityAppearedPacket {
     #[length(24)]
     pub name: String,
 }
+#[derive(Debug, Clone, Packet, ServerPacket, MapServer)]
+#[cfg_attr(feature = "interface", derive(korangar_interface::elements::PrototypeElement))]
+#[header(0x0148)]
+pub struct ResurrectionPacket {
+    pub entity_id: EntityId,
+    pub p_type: u16,
```

Comment:
What is `p_type`?

Reply:
the packet type, i search rathena source code and the only possible value is 0.

Reply:
Then please rename the field to `packet_type` and set the `#[new_default]` attribute. A comment saying `/// Alywas 0 on rAthena` would also be appreciated